src/middlewared/middlewared/plugins/vm/vms.py

```diff
@@ -112,6 +114,7 @@ async def extend_vm(self, vm, context):
         Str('cpuset', default=None, null=True, validators=[NumericSet()]),
         Str('nodeset', default=None, null=True, validators=[NumericSet()]),
         Bool('pin_vcpus', default=False),
+        Bool('pause_on_snapshot', default=False),
```

Comment:
You use `suspend` for all internal function names and also use `suspend_vms` as the schema arg in the zfs plugin but this schema arg uses `pause`. Can we change this to `suspend_on_snapshot`?